cl_manycore/libraries/bsg_manycore_loader.cpp

```diff
@@ -946,3 +946,40 @@ int hb_mc_loader_symbol_to_eva(const void *bin, size_t sz, const char *symbol,
 
         return HB_MC_SUCCESS;
 }
+
+
+int hb_mc_loader_read_program_file(const char *file_name, unsigned char **file_data, size_t *file_size)
```

Comment:
Can you add a comment describing the purpose of this function?

Reply:
My bad, forgot. Fixed, thanks!